server/svix-server/src/core/operational_webhooks.rs

```diff
@@ -25,47 +25,47 @@ use crate::{
 /// Sent when an endpoint has been automatically disabled after continuous failures.
 #[derive(Debug, Serialize)]
 #[serde(rename_all = "camelCase")]
-pub struct EndpointDisabledEvent<'a> {
-    pub app_id: &'a ApplicationId,
-    pub app_uid: Option<&'a ApplicationUid>,
-    pub endpoint_id: &'a EndpointId,
-    pub endpoint_uid: Option<&'a EndpointUid>,
+pub struct EndpointDisabledEvent {
+    pub app_id: ApplicationId,
+    pub app_uid: Option<ApplicationUid>,
+    pub endpoint_id: EndpointId,
+    pub endpoint_uid: Option<EndpointUid>,
     pub fail_since: DateTime<Utc>,
 }
 
 /// Sent when an endpoint is created, updated, or deleted
 #[derive(Debug, Serialize)]
 #[serde(rename_all = "camelCase")]
-pub struct EndpointEvent<'a> {
-    pub app_id: &'a ApplicationId,
-    pub app_uid: Option<&'a ApplicationUid>,
-    pub endpoint_id: &'a EndpointId,
-    pub endpoint_uid: Option<&'a EndpointUid>,
+pub struct EndpointEvent {
+    pub app_id: ApplicationId,
+    pub app_uid: Option<ApplicationUid>,
+    pub endpoint_id: EndpointId,
+    pub endpoint_uid: Option<EndpointUid>,
 }
 
-impl<'a> EndpointEvent<'a> {
-    pub fn new(app_uid: Option<&'a ApplicationUid>, endp: &'a endpoint::Model) -> Self {
+impl EndpointEvent {
+    pub fn new(app_uid: Option<&ApplicationUid>, endp: &endpoint::Model) -> Self {
         Self {
-            app_id: &endp.app_id,
-            app_uid,
-            endpoint_id: &endp.id,
-            endpoint_uid: endp.uid.as_ref(),
+            app_id: endp.app_id.clone(),
+            app_uid: app_uid.map(Clone::clone),
```

Comment:
Can we change this (and other occurrences) to used `.cloned` instead of `.map`? IMO is a bit cleaner.

Reply:
TIL about that that method. Agreed it's much cleaner, should be updated now.